test_scripts/RC/Capabilities/006_Processing_of_requests_with_unsupported_params.lua

```diff
@@ -0,0 +1,334 @@
+      fanSpeed = 30  -- absent
+    }
+  },
+  hmiRequest = {
+    moduleType = "CLIMATE",
+    climateControlData =
+    {
+      acEnable = true, -- OK
+      dualModeEnable = true, --OK
+      -- fanSpeed = 30  -- absent
+    }
+  },
+  hmiResponse = {
+    moduleType = "CLIMATE",
+    climateControlData =
+    {
+      acEnable = true, -- OK
+      dualModeEnable = true, --OK
+      -- fanSpeed = 30  -- absent
+    }
+  },
+  mobileResponse = {
+    moduleType = "CLIMATE",
+    climateControlData =
+    {
+      acEnable = true, -- OK
+      dualModeEnable = true, --OK
+      -- fanSpeed = 30  -- absent
+    }
+  },
+  info = "fanSpeedAvailable is not available on HMI"
+}
+
+local param2ok0nok = { -- resultCode: SUCCESS, success: true
+  mobileRequest = {
+    moduleType = "CLIMATE",
+    climateControlData =
+    {
+      acEnable = true, -- OK
+      dualModeEnable = true --OK
+    }
+  },
+  hmiRequest = {
+    moduleType = "CLIMATE",
+    climateControlData =
+    {
+      acEnable = true, -- OK
+      dualModeEnable = true --OK
+    }
+  },
+  hmiResponse = {
+    moduleType = "CLIMATE",
+    climateControlData =
+    {
+      acEnable = true, -- OK
+      dualModeEnable = true --OK
+    }
+  },
+  mobileResponse = {
+    moduleType = "CLIMATE",
+    climateControlData =
+    {
+      acEnable = true, -- OK
+      dualModeEnable = true --OK
+    }
+  },
+  info = nil
+}
+
+local param0ok1nok1dis = { -- resultCode: UNSUPPORTED_RESOURCE, success: false
+  mobileRequest = {
+    moduleType = "CLIMATE",
+    climateControlData =
+    {
+      acMaxEnable = true, -- NOK
+      fanSpeed = 30  -- absent
+    }
+  },
+  hmiRequest = nil,
+  hmiResponse = nil,
+  mobileResponse = nil,
+  info = nil
+}
+
+local param1ok1nok1dis = { -- resultCode: UNSUPPORTED_RESOURCE, success: true
+  mobileRequest = {
+    moduleType = "CLIMATE",
+    climateControlData =
+    {
+      acEnable = true, -- OK
+      acMaxEnable = true, -- NOK
+      fanSpeed = 30  -- absent
+    }
+  },
+  hmiRequest = {
+    moduleType = "CLIMATE",
+    climateControlData =
+    {
+      acEnable = true, -- OK
+      -- acMaxEnable = true, -- NOK
+      -- fanSpeed = 30  -- absent
+    }
+  },
+  hmiResponse = {
+    moduleType = "CLIMATE",
+    climateControlData =
+    {
+      acEnable = true, -- OK
+      -- acMaxEnable = true, -- NOK
+      -- fanSpeed = 30  -- absent
+    }
+  },
+  mobileResponse = {
+    moduleType = "CLIMATE",
+    climateControlData =
+    {
+      acEnable = true, -- OK
+      -- acMaxEnable = true, -- NOK
+      -- fanSpeed = 30  -- absent
+    }
+  },
+  info = "acMaxEnableAvailable, fanSpeedAvailable is not available on HMI"
+}
+
+--[[ Local Functions ]]
+local function getHMIParams()
+  local function getHMICapsParams()
+    if fullSupportedModule == "CLIMATE" then
+      return { commonRC.DEFAULT, capabilities, commonRC.DEFAULT }
+    elseif fullSupportedModule == "RADIO" then
+      return { capabilities, commonRC.DEFAULT, commonRC.DEFAULT }
```

Comment:
@Itileda moduleName is just string with length less than 100 characters. Exists no requirements about its value.
Current case use only climate capabilities. In case it be appended with Radio capabilities will be used moduleName like something about radio.